tree/master/packages/extension/public/locales/pl/translation.json

```
@@ -0,0 +1,162 @@
+	  "I want to forget this account": "Chcę zapomnieć to konto",
+	  "Scan Address Qr": "Skanuj Adres Qr",
+	  "Add the account with identified address": "Dodaj konto ze zidentyfikowanym adresem",
+	  "Import account": "Importuj konto",
+	  "existing 12 or 24-word mnemonic seed": "istniejący 12 lub 24-wyrazowy ziarno mnemoniczne",
+	  "Add the account with the supplied seed": "Dodaj konto z dostarczonym materiałem seedem",
+	  "Metadata": "Metadane",
+	  "from": "od",
+	  "chain": "łańcuch",
+	  "icon": "ikona",
+	  "decimals": "dziesiętne",
+	  "symbol": "symbol",
+	  "upgrade": "aktualizacja",
+	  "This approval will add the metadata to your extension instance, allowing future requests to be decoded using this metadata.": "To zatwierdzenie doda metadane do instancji wtyczki, umożliwiając dekodowanie przyszłych żądań przy użyciu tych metadanych.",
+	  "Yes, do this metadata update": "Tak, zaktualizuj metadane aktualizacje",
+	  "Reject": "Odrzuć",
+	  "Restore from JSON": "Przywróć z JSON",
+	  "backup file": "plik kopii zapasowej",
+	  "Password for this file": "Hasło do tego pliku”,
+	  "Restore": "Przywróć",
+	  "bytes": "bajty",
+	  "method data": "dane metody",
+	  "method": "metoda",
+	  "info": "informacje",
+	  "immortal": "nieśmiertelny",
+	  "mortal, valid from {{birth}} to {{death}}": "śmiertelna, ważna od {{birth}} do {{death}}",
+	  "genesis": "geneza",
+	  "version": "wersja",
+	  "nonce": "nonce",
+	  "tip": "napiwek",
+	  "Sign the transaction": "Podpisz transakcję",
+	  "Sign the message": "Podpisz wiadomość",
+	  "Transaction": "Transakcja",
+	  "Sign message": "Podpisz wiadomość",
+	  "Scan signature via camera": "Skanuj podpis za pomocą kamery",
+	  "Don't ask me again for the next {{expiration}} minutes": "Nie pytaj mnie ponownie przez następne {{expiration}} minut",
+	  "Password for this account": "Hasło do tego konta",
+	  "Welcome": "Witamy",
+	  "Before we start, just a couple of notes regarding use:": "Zanim zaczniemy kilka uwag dotyczących użytkowania:",
+	  "We do not send any clicks, pageviews or events to a central server": "Nie wysyłamy żadnych kliknięć, odsłon ani zdarzeń na centralny serwer",
+	  "We do not use any trackers or analytics": "Nie używamy żadnych narzędzi śledzących ani analiz",
+	  "We don't collect keys, addresses or any information - your information never leaves this machine": "Nie zbieramy kluczy, adresów ani żadnych informacji - Twoje informacje nigdy nie opuszczają tego komputera",
+	  "... we are not in the information collection business (even anonymized).": "... nie zajmujemy się zbieraniem informacji (nawet anonimowymi).",
+	  "Understood, let me continue": "Zrozumiałem, kontynuuję",
+	  "An error occured": "Błąd wystąpił ",
```

```suggestion
	  "An error occured": "Wystąpił błąd ",
```
Chyba, że z kontekstu lepiej brzmi odwrotnie.